extensions/typescript/package.json

```
@@ -73,6 +77,22 @@
         "actionItem": {
           "label": "${config.typescript.showExternalReferences && \"Hide other repositories\" || \"Show other repositories\"}"
         }
+      },
+      {
+        "id": "supplementReferencesWithSearchResults.toggle",
+        "command": "updateConfiguration",
+        "title": "${!!config.codeIntel.supplementReferencesWithSearchResults && \"Hide search results\" || \"Show search results\"}",
```

I think it would be good to consistently stick with "search-based"
```suggestion
        "title": "${!!config.codeIntel.supplementReferencesWithSearchResults && \"Hide search-based results\" || \"Show search-based results\"}",
```

---

I think we need something better than either of these. The text should actually read "stop showing search-based results if there are precise ones" but I don't know a very clear and succinct way to say that as a text link.